data/json/items/generic.json

```diff
@@ -2730,7 +2734,7 @@
   {
     "id": "clay_pot_flower",
     "type": "GENERIC",
-    "category": "tools",
+    "category": "other",
```

Comment:
Putting stuff back into "other" seems like a step backwards.  They don't seem to have any uses, so "salvage" instead?  Alternately, "construction materials", "containers", or "deployables".

Reply:
The categories are not in yet and this bugs me.